module/VuFindTheme/src/VuFindTheme/View/Helper/ConcatTrait.php

```diff
@@ -368,6 +368,7 @@ protected function outputInOrder($indent)
         $output = [];
         foreach ($this->groups as $group) {
             if (isset($group['other'])) {
+                /* @phpstan-ignore-next-line */
```

Comment:
Do we have to ignore this because phpstan gets confused by calls from a Trait to methods undefined inside that Trait? Are we unable to use `@method` annotations to work around this? I just want to be absolutely sure there's no better option before we resort to ignore annotations.

Reply:
No, phpstan could analyse traits in context of classes using them. There is problem with incompatible signature of `itemToString` methods in HeadingLink and HeadingScript view helpers - one versus four arguments.

So it could probably be fixed, but with some refactoring of the trait and/or view helpers.

Reply:
Ahh, I see! Maybe a compromise would be to add a comment explaining why we're ignoring the next line, so we can address it in the future... e.g.:

```
/**
 * PHPStan doesn't like this because of incompatible itemToString signatures in HeadLink/HeadScript,
 * but it is safe to use because the extra parameters will be ignored appropriately.
 *
 * @phpstan-ignore-next-line
 */
```

Obviously, refactoring to solve the problem would be even better, but I can see how that might get complicated, and changing the base HeadLink/HeadScript helpers is not an option since they're coming from the framework.

Reply:
Thanks @demiankatz. I added the comment.